index.Rmd

```diff
@@ -23,3 +23,6 @@ knitr::opts_chunk$set(echo = TRUE)
 knitr::include_graphics("https://learning.nceas.ucsb.edu/images/NCEAS_logo.png")
 ```
 
+# Additional Information
+
```

Comment:
I'd rather this be a subsection.

Reply:
okay I can make that change

Reply:
Alright then I'll merge when you send that.